web_src/js/components/DashboardRepoList.vue

```diff
@@ -464,7 +464,7 @@ const sfc = {
           }
           break;
       }
-      if (this.activeIndex === -1 || this.activeIndex > this.repoTypeCount - 1) {
+      if (this.activeIndex === -1 || (this.repoTypeCount > 0 && this.activeIndex > this.repoTypeCount - 1)) {
```

Comment:
Actually, the `this.activeIndex > this.repoTypeCount - 1)` check shouldn't be here.

It should be in `searchRepos`, after the `count` changes.

Reply:
And `repoTypeCount` doesn't seem the right "count" to be checked, IIRC "repoTypeCount" means the total count of current type. But I guess you want to check the count of repo items listed on current page.

Reply:
Yes, but I don't know how to get count after searchRepos inside keydown event

Reply:
Hi @wxiaoguang ,

Do you know how to get count data after searchRepos completes? Similar to using jquery ajaxStop?

Thanks

Reply:
The count is `this.repos.length`, it is assigned in `searchRepos`, after the `await`.